### PR TITLE
3 gallery page

### PR DESCRIPTION
**Pull request by IgorSinyak**
gallery page added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<html lang="en" ng-app="wireframeApp">
+  <head>
+    <meta charset="UTF-8">
+    <title>Wireframe</title>
+    <link rel="stylesheet" href="styles/main.css">
+    <link rel="stylesheet" href="bower_components/font-awesome/css/font-awesome.min.css">
+  </head>
+  <body>
+    <div class="wrapper">
+      <div ui-view="header"></div>
```
Comment (vitali-usik) [inline index.html:11]: You can use element `<header>` and `footer`

File: scripts/main.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+$(document).ready(function() {
```
Comment (vitali-usik) [inline scripts/main.js:1]: Are you sure about using jQuery in this way? Doesn't angular do the same thing?

File: scripts/services.js [hunk 3]
```diff
@@ -0,0 +1,197 @@
+'use strict';
+
+angular.module('wireframeApp')
+  .factory('guitarListFactory', function() {
+
+    var guitarsFactory = {};
+
+    var guitars = [
```
Comment (vitali-usik) [inline scripts/services.js:8]: Move to separate const js-file or json

File: scripts/main.js (hunk 2 above)
Comment (IgorSinyak) [reply scripts/main.js:1]: This file don`t need here, i ll delete it. It was only experiment=)

**Pull request closed without merging**